examples/dt/neutron/3cont_2comp_3net-ipv4-geneve-composable_provider_network

```diff
@@ -0,0 +1,66 @@
+# Deployed Topology 3cont_2comp_3net-ipv4-geneve-composable-vlan-provider-network
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker                                  | vm    | 3      |
+| CFME tester                                | vm    | 1      |
```

Comment:
IMO, configuration of CFME tester node (which is actually a router node supporting public network of type 'vlan') can be a  separate stage in the list of stages below. At least with jenkins+infrared we had a separate stage for configuring an external router. Probably with moving to OCP infrastructure we can just configure networks accordingly and get rid of this node, but not 100% sure.

Reply:
Separate stage added